src/shared/components/comment/comment-node.tsx

```diff
@@ -332,6 +335,7 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
                   <Icon icon="lock" classes="icon-inline text-danger" />
                 </span>
               )}
+              {deleted_or_removed && <Icon icon="trash" classes="ms-1" />}
```

Comment:
Why not just copy the block right above this instead of writing from scratch. There's an example with the correct classes, and a tippy right above this.

Reply:
Made it more consistent and updated the screenshot above. Using red icon seemed to strong so I removed that. Also marked the reply button as disabled if the comment is deleted or locked.